common/p_map.cpp

```diff
@@ -3116,6 +3116,9 @@ msecnode_t *P_AddSecnode (sector_t *s, AActor *thing, msecnode_t *nextnode)
 
 	msecnode_t *node;
 
+	if (s == 0)
```

Comment:
Can you change to "s == NULL" to more clearly convey this is a check for a null pointer?

Reply:
I just made the change with my latest commit.